class-work/02-deploy-python.qmd

```diff
@@ -62,21 +62,25 @@ vetiver.model_card()
 ```{python}
 from vetiver import VetiverAPI
 
-api = __
+api = ___
 api.run()
 ```
 
 ## Deploy to Posit Connect 🚀
 
 ```{python}
 from rsconnect.api import RSConnectServer
+import os
+api_key = os.getenv("CONNECT_API_KEY")
+rsc_url = os.getenv("CONNECT_SERVER")
+
 connect_server = RSConnectServer(url = rsc_url, api_key = api_key)
-board = pins.board_connect(server_url = url, api_key = api_key, allow_pickle_read = True)
+board = pins.board_connect(server_url = rsc_url, api_key = api_key, allow_pickle_read = True)
```

Comment:
This change is needed in the slides too.

Or we could (in both places) do:

```suggestion
board = pins.board_connect(allow_pickle_read = True)
```

I know this doesn't save us a ton, but we haven't really shown these args anywhere else using a Connect board.